cmd/up.go

```diff
@@ -27,10 +28,10 @@ func UpCommand() cli.Command {
 	}
 }
 
-func ClusterUp(clusterFile string) (string, string, string, string, error) {
+func ClusterUp(clusterFile string, customDialer hosts.CustomDialer) (string, string, string, string, error) {
```

Comment:
doesn't look like all the values this method return, being used? why not just return an error then?

Reply:
these are the values that kontainer-engine requested like `APIURL`,`cacrt`,`cakey`, etc

Reply:
Unless they are used when called from kontainer-engine?